src/app/app-routing.module.ts

```diff
@@ -1,7 +1,27 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomePageComponent } from './modules/home-page/components/home-page/home-page.component';
+import { PointsShopPageComponent } from './modules/pointsshop-page/components/pointsshop-page/pointsshop-page.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+	{
+		path: 'index',
+		component: HomePageComponent
+	},
+	{
+		path: '',
+		pathMatch: 'full',
+		redirectTo: 'index'
+	},
+	{
+		path: 'pointsshop',
+		component: PointsShopPageComponent
+	},
+	// {
+	// 	path: '**',
+	// 	redirectTo: 'index'
+	// }
```

Comment:
If you don't want to remove some code part by any reason, you should leave a `// TODO: talk-talk-talk` comment. Otherwise this code looks like unused and must be removed! :smiling_imp: